dockerize.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./gradlew
+docker build -t fizzbuzz-enterprise-edition .
+docker run -it --rm --name fizzbuzz-enterprise-edition fizzbuzz-enterprise-edition
```

Comment:
We require the use of long form arguments for documentation purposes and all dockerized applications container images must be clearly namespaced. Please correct these discrepancies before this git pull request can be merged.
```
docker build \
  --rm \
  --tag enterprisequalitycoding/fizzbuzz-enterprise-edition \
  --file Dockerfile \
  .

docker run  \
  --interactive \
  --tty \
  --rm \
  --name fizzbuzz-enterprise-edition \
  enterprisequalitycoding/fizzbuzz-enterprise-edition
```